src/db_impl.cc

```diff
@@ -269,6 +270,20 @@ Status TitanDBImpl::OpenImpl(const std::vector<TitanCFDescriptor>& descs,
         blob_file_set_.get(), stats_.get()));
     cf_opts.table_factory = titan_table_factories.back();
     cf_opts.merge_operator = shared_merge_operator_;
+    if (cf_opts.compaction_filter != nullptr ||
```

Comment:
We need to update the same logic to `CreateColumnFamilies()`

Reply:
Done